Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py

```diff
@@ -1145,11 +1145,14 @@ def parse_incident_from_item(item, is_fetch):  # pragma: no cover
                                     if (header.name, header.value) not in attached_email_headers \
                                             and header.name != 'Content-Type':
                                         attached_email.add_header(header.name, header.value)
-
+                            try:
+                                formatted_message = attached_email.as_string()
+                            except UnicodeEncodeError:
+                                formatted_message = attached_email.as_bytes()  # type: ignore[assignment]
```

Comment:
why not change the type hint for formatted_message in get_attachment_name instead of adding this mypy type ignore?